### ISSUE TITLE
test suite compilation failure

### ISSUE THREAD

**Issue by bergmark** [mongodb-haskell/mongodb#61]
```
Preprocessing test suite 'test' for mongoDB-2.1.0...
[1 of 1] Compiling Main             ( test/Main.hs, dist/build/test/test-tmp/Main.o )

test/Main.hs:8:1: error:
    Failed to load interface for ‘Spec’
    Use -v to see a list of the files searched for.
```

**Comment by bergmark** [comment]
Got this while building stackage nightly. If you want stackage to run the test, please send a pull request to re-enable the test-suite when it's fixed.

**Comment by VictorDenisov** [comment]
@bergmark I'm looking into this.

**Comment by VictorDenisov** [comment]
@bergmark I released 2.1.0.1 version which fixes the compiler error, but if stackage actually runs the test then it will fail because the test requires a running instance of mongodb server. I presume it's ok to keep it as is. I also posted another pull request with another missing part: https://github.com/fpco/stackage/pull/1649
